initialization/initialization_test.go

```diff
@@ -55,9 +57,45 @@ func TestInitialize(t *testing.T) {
 		r.NoError(init.Initialize(ctx))
 		cancel()
 		eg.Wait()
-
-		r.Equal(uint64(cfg.MinNumUnits)*cfg.LabelsPerUnit, init.SessionNumLabelsWritten())
 	}
+	r.Equal(uint64(cfg.MinNumUnits)*cfg.LabelsPerUnit, init.SessionNumLabelsWritten())
+
+	r.NotNil(init.nonce)
+	r.NoError(verifying.VerifyPoW(*init.nonce, uint64(cfg.MinNumUnits), uint64(cfg.BitsPerLabel), init.commitment))
+}
+
+func TestInitialize_PowOutOfRange(t *testing.T) {
```

Comment:
Add asserts to other tests to verify found nonce

Add test that stops initializing with and without nonce